pkg/util/scrubber/default_test.go

```diff
@@ -285,53 +285,65 @@ func TestSNMPConfig(t *testing.T) {
 		`   community_string: ********`)
 	assertClean(t,
 		`
-snmp_traps_config:
-  community_strings:
-    - 'password1'
-    - 'password2'
+network_devices:
```

Comment:
I think we should still keep one/two tests for older `snmp_traps_config.community_strings` to make sure we maintain backward compatibility for scrubbing community string set with `snmp_traps_config.community_strings`.

The cost is low to do so and can avoid leaking community strings (e.g. users still having legacy config in newer versions of Agent).

Reply:
Good idea, I did not think of this 👍